install-cluster/README.adoc

```diff
@@ -0,0 +1,85 @@
+= Install kubernetes cluster on AWS using kops
+:toc:
+
+This tutorial will walk you through how to install kubernetes cluster using kops.
+
+. Install kops and kubectl on your PC
+
+    brew update && install kops
+    curl -LO https://storage.googleapis.com/kubernetes-release/release/$(curl -s https://storage.goo\
+    gleapis.com/kubernetes-release/release/stable.txt)/bin/darwin/amd64/kubectl
+    chmod +x ./kubectl
+    sudo mv ./kubectl /usr/local/bin/kubectl
+
+. IAM user permission
+
+    Make sure the latest version of http://docs.aws.amazon.com/cli/latest/userguide/installing.html[AWS CLI]
+    is installed. User permissions being used must have this IAM capabilities
```

Comment:
Add the exact CLI command to enable these permissions?

Reply:
I don't think we need CLI command here bcoz it will become lengthy and not the focus of this doc. These are IAM policies. I added link to IAM policies